packages/test/snapshots/src/replayMultipleFiles.ts

```diff
@@ -77,7 +79,15 @@ class ConcurrencyLimiter {
     }
 }
 
-export async function processOneFile(args: IWorkerArgs) {
+/**
+ * Processes one node of the snapshot contents folder. Each node has the following:
+ * 1. messages.json - This file contains the messages to be replayed by the replay tool.
+ * 2. A list of snapshots in the current version. When running in "Compare" mode, snapshots generated by the replay
```

Comment:
by compare here do you mean back compat like the enum value below?

Reply:
I mean the Compare enum value. Actually this comment is outdated now that the folder structure has changed. I will update it.